packages/styles/dialog.css

```
@@ -100,3 +100,41 @@
   border-top: 1px solid var(--dialog-border-color);
   padding: calc(var(--space-small) - 4px) var(--dialog-padding);
 }
+
+/**
+* style for plain dialog
+*/
+
+.Modal--info .Dialog__header {
+  display: flex;
+  justify-content: center;
+  border: none;
+  background-color: white;
```

```suggestion
  background-color: transparent;
```

The current value of `white` would be layered on top of any custom background colors.